python/python/tests/test_async_client.py

```diff
@@ -2123,9 +2126,11 @@ async def test_zrangestore_by_score(self, redis_client: TRedisClient):
             )
             == 2
         )
-        assert await redis_client.zrange_withscores(
+
```

Comment:
Why did you add here empty line?
In prev changes, you put it W/O.
Please be consistent with it in all places.